gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -228,8 +228,8 @@ <h3 class="mdc-layout-grid__cell--span-3">
   the page is loaded. So we decide to programmatically click on header of collapsed
   packages after the page is loaded, and the content will be visible.
   */
-  window.addEventListener('load', function() {
-    const package_headers = document.querySelectorAll('.package-header[affordance="collapse"]')
+  document.addEventListener('turbo:load', function() {
+    const package_headers = document.querySelectorAll('.package-header[affordance="collapse"][aria-expanded="false"]')
```

Comment:
`turbo:load` happens multiple times but we just want to click the headers once, so only click them when `aria-expanded` is false